config/locations.yml

```diff
@@ -5,94 +5,95 @@ aginternet:
   name: "Alberta Government Library - Internet"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
 aglcap:  
-  name: "Alberta Government Library"
+  name: "Alberta Government Library - Capital Boulevard"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
```

Comment:
This is the location listed in the url.  Not sure if this should be applied to all the other 'Alberta Government Library' locations that weren't previously included.